activestorage/test/test_helper.rb

```diff
@@ -174,6 +174,8 @@ class User < ActiveRecord::Base
   has_one_attached :avatar_with_variants do |attachable|
     attachable.variant :thumb, resize_to_limit: [100, 100]
   end
+  has_one_attached :intro_video
+  has_one_attached :name_pronunciation_audio
```

Comment:
Those are not strictly necessary since we could use `user.avatar` with the same result, but I added them to avoid confusion on tests.